scripts/README.md

```diff
@@ -0,0 +1,21 @@
+# UniOne API test script
```

Comment:
We don't need so detailed description, because the script is for internal usage. Keep it simple:

Usage:

1) `compose require unione/unione-php`
2) `cd vendor/unione/unione-php && composer test {YOUR-HOST-NAME} {YOUR-API-KEY}`

You should copy `example.config.php` to `config.php` file and set your parameters.

After that, specify that it returns `0` on success and `1` in case of failure.